### ISSUE TITLE
Documentation missing on px-data-table-column 

### ISSUE THREAD

**Issue by swapnavad** [PredixDev/px-data-table#108]
# Submit an Issue

Hi,

Thanks for helping us improve the Predix UI platform by submitting an issue.

Before you begin, please check the list of existing issues to avoid submitting a duplicate issue.

To help us solve this issue as fast as possible, please fill out the following sections:

## Expected behavior and actual behavior:
Missing documentation on all attribute ,example "hide".  

## Code examples help us better understand the issue - follow the appropriate codepen for the component by going to https://predix-ui.com, finding the component in question, and clicking on the pencil icon under the demo.
        Once you've created your code example, you can save it under a new url.
        Please note that you should NOT use the same methods for production as the ones used in codepen - these methods are not production ready.
  <px-data-table-column name="Status" hide="true"></px-data-table-column>

**Comment by randyaskin** [comment]
Go to https://www.predix-ui.com/#/components/px-data-table/ and use the dropdown at the top of the screen to switch to the child component, px-data-table-column. The API is there.

**Comment by swapnavad** [comment]
Hi Randy , 
Thank you for your response.This is the usability issue on this UI component, I also tried to search for px-data-table-column on the search box and this item does not exist on the navigation bar too. I did not even notice the top dropdown for documents. 

I think it will be useful in parent page for px-data-table , should have some verbatim( maybe just how to locate them ) about the documentation for these child components. 
The parents page is showing an example to show how to use the child component, maybe that can be a location to mentioned how to look up for the documentation.
